pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computefirewallpolicyrule/computefirewallpolicyrule-egress-full/_http.log

```diff
@@ -905,7 +803,7 @@ X-Xss-Protection: 0
     ]
   },
   "priority": 9000,
-  "ruleTupleCount": 109,
+  "ruleTupleCount": 4,
```

Comment:
the `ruleTupleCount` is generated, it's used to determine the complexity of a rule. To make it easier, in the mock server, if a rule is more complicated, I assign it to 4, otherwise to 2. So this diff is expected.